docs/src/docs/01-index.md

```diff
@@ -1,18 +1,13 @@
 ---
-title: cloud.gov design system
+title: Overview
```

Comment:
Since this is the first thin you see, it might be good to have this be something like "cloud.gov Pattern Library" and Overview as the first header.